src/main/charts/agent/values.yaml

```diff
@@ -0,0 +1,233 @@
+##
+
+# -- The initial number of Bamboo agent pods that should be started at deployment time. 
+#
+replicaCount: 1
+
+# Image configuration 
+#
+image:
+
+  # -- The Bamboo agent Docker image to use
+  # https://hub.docker.com/r/atlassian/bamboo-agent-base
+  #
+  # TODO: This needs to be updated when docker porting work is complete
+  repository: atlssmith/bamboo-agent-base 
+  
+  # -- Image pull policy
+  #
+  pullPolicy: IfNotPresent
+  
+  # -- The docker image tag to be used - defaults to the Chart appVersion
+  #
+  tag: ""
+  
+# K8s ServiceAccount configuration. Give fine-grained identity and authorization 
+# to Pods
+#
+serviceAccount:
+
+  # -- Set to 'true' if a ServiceAccount should be created, or 'false' if it 
+  # already exists.
+  #
+  create: true
+  
+  # -- The name of the ServiceAccount to be used by the pods. If not specified, but 
+  # the "serviceAccount.create" flag is set to 'true', then the ServiceAccount name 
+  # will be auto-generated, otherwise the 'default' ServiceAccount will be used.
+  # https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server
+  #
+  name:
+  
+  # -- For Docker images hosted in private registries, define the list of image pull 
+  # secrets that should be utilized by the created ServiceAccount
+  # https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  #
+  imagePullSecrets: []
+  # - name: secretName
+
+# Bamboo agent configuration 
+#
+agent:
+  
+  # The security token with which the agent will authenticate to the Bamboo server
+  # Additional details here:
+  # https://confluence.atlassian.com/bamboo/agent-authentication-289277196.html#Agentauthentication-SecuritytokenverificationSecuritytokenverification
+  #
+  securityToken:
+    
+  # The K8s DNS record for the Bamboo server service. Should be of the form, <namespace>.<service_name>.svc.cluster.local
+  # for example: "bamboo.bamboo.svc.cluster.local"
+  # 
+  # This value can be obtained by running:
+  # export NAMESPACE=<namespace in which the Bamboo server resides>
+  # echo "$(echo "${NAMESPACE}.")""$(kubectl exec -it bamboo-0 -- cat /etc/resolv.conf | grep search | awk '{print $2}')"
+  # 
+  server:
+    
+  # Enable or disable security context in StatefulSet template spec. Enabled 
+  # by default with UID 2005. Disable when deploying to OpenShift, unless anyuid 
+  # policy is attached to a service account.
+  #
+  securityContext:
+  
+    # -- Set to 'true' to enable the security context
+    #
+    enabled: true
+    
+    # -- The GID used by the Bamboo agent docker image
+    #
+    gid: "2005"
+
+  # Port definitions
+  #
+  ports:
+  
+    # -- The port on which the Bamboo agent listens for HTTP traffic
+    #
+    http: 80
+     
+  # TODO: As of yet a /status endpoint does not exist for agents. Once delivered the implementation for this
```

Comment:
Bamboo agents to not have a /status endpoint yet. Ticket created to be actioned by Bamboo team:
https://bulldog.internal.atlassian.com/browse/BAMBK8S-84